common.sh

```diff
@@ -20,9 +20,6 @@
 set -ex
 set -o pipefail
 
-_registry_port="5001"
```

Comment:
en.... I don't like force port number. I hope to keep it flexible.

Reply:
The reason why I chose to fix the port here is that if remain such flexibility, then we have a very ugly sed scripts to replace the port info inside the local-registry.yml.
As you could see in current PR diff, I also cleaned up there to make code cleaner:)
Flexibility often comes with complexity:)  So we may do trade-off. 
Some of the options are not so important to users, such as the registry port, for local-dev use and CI use, IMO, fix to 5001 is acceptable.

Reply:
if we have to flexible with port number, any better solution than sed?

Reply:
> Maybe we can use ex command like this:
> 
> ```
> ex -s -c "1,\$s/$kind_registry_name/${KIND_REGISTRY_NAME}/ge" -c "x" "${KIND_DIR}"/kind.yml
> ```
> 
> I think it is available on Mac and Linux


In my dev machine, the `ex` points to the same binary as `vim` :-)
$ ls -l /usr/bin/ex
lrwxrwxrwx 1 root root 20 Aug 24  2021 /usr/bin/ex -> /etc/alternatives/ex
$ ls -l /usr/bin/vim
lrwxrwxrwx 1 root root 21 Aug 24  2021 /usr/bin/vim -> /etc/alternatives/vim
$ ls -l /etc/alternatives/ex
lrwxrwxrwx 1 root root 18 Aug 24  2021 /etc/alternatives/ex -> /usr/bin/vim.basic
$ ls -l /etc/alternatives/vim
lrwxrwxrwx 1 root root 18 Aug 24  2021 /etc/alternatives/vim -> /usr/bin/vim.basic

Reply:
Yes, it's the line editor for [vim](https://www.computerhope.com/unix/uex.htm#:~:text=On%20Unix%2Dlike%20operating%20systems,program%20written%20by%20Charles%20Haley.) 😅
It was new to me as well

Reply:
em... well technologically maybe sed still one of best options?
I mean just when we replace string via shell.

Reply:
This issue is fixed in #31 by using `sed < manifest/yaml > generated/yaml`